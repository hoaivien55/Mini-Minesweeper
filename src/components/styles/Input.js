import styled from "styled-components";

const Input = styled.input`
    width: 100%;
	border-radius: 5px;
	border: 1px solid #727a7045;
	font-size: 20px;
	padding: 5px;
	outline: none;
    transition: box-shadow 0.5s;
	&:hover {
		box-shadow: 0.2rem 0.8rem 1.6rem #d9d9d9;
	}
    &[type="submit"] {
        color: red;
        background: #a4f5fd24;
        cursor: pointer;
        font-weight: bold;
    }
`;

export default Input;
