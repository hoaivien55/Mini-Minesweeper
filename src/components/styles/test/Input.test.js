import renderer from "react-test-renderer";
import Input from "../Input";

it("Input: Virtual DOM snapshot", () => {
	const tree = renderer.create(<Input />).toJSON();
	expect(tree).toMatchSnapshot();
});
