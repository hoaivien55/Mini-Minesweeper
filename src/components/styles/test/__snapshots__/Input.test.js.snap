// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input: Virtual DOM snapshot 1`] = `
.c0 {
  width: 100%;
  border-radius: 5px;
  border: 1px solid #727a7045;
  font-size: 20px;
  padding: 5px;
  outline: none;
  -webkit-transition: box-shadow 0.5s;
  transition: box-shadow 0.5s;
}

.c0:hover {
  box-shadow: 0.2rem 0.8rem 1.6rem #d9d9d9;
}

.c0[type="submit"] {
  color: red;
  background: #a4f5fd24;
  cursor: pointer;
  font-weight: bold;
}

<input
  className="c0"
/>
`;
